subscriptions/tasks.py

```diff
@@ -327,23 +378,15 @@ def run(self, subscription_id, **kwargs):
         try:
             subscription = Subscription.objects.get(id=subscription_id)
             if subscription.process_status == 0:
-                scheduler = self.scheduler_client()
-                # get the messageset length for frequency calc
-                message_count = subscription.messageset.messages.filter(
-                    lang=subscription.lang).count()
-                # next_sequence_number is for setting non-one start position
-                # frequency is the number of messages that should be sent
-                frequency = (
-                    message_count - subscription.next_sequence_number) + 1
-                # Build the schedule POST create object
                 schedule = {
-                    "frequency": frequency,
+                    "frequency": None,
```

Comment:
We're not calculating it any more. The purpose here is to not calculate the number of times the scheduler should run when the subscription is created. Instead it should just be deactivated when it reaches the last message in the messageset.  The problem is if we set the number of times the schedule should run when the subscription is created, then if someone for instance adds one additional message to the message set, the scheduler will stop before the message set has finished. Then you'll never move on to the next message set.